hardware/qcom/Android.mk

```diff
@@ -35,9 +38,6 @@ TARGET_KERNEL_VERSION := $(SOMC_KERNEL_VERSION)
 
 include device/sony/common/hardware/qcom/utils.mk
 
-include $(display-hal)/Android.mk
 include $(call all-makefiles-under,$(audio-hal))
-include $(call first-makefiles-under,$(ipa-hal))
 include $(call all-makefiles-under,$(gps-hal))
-include $(call all-makefiles-under,$(media-hal))
```

Comment:
If these are removed, why not drop the now-unused variables above too?

Reply:
because i saw there are other seemingly unused variables in there prior to this change. that made me assume these are somehow used elsewhere, and no i did not check :D

Reply:
The lowercase vars aren't used anywhere else IIRC, but the uppercase vars are being referred from some HALs theselves.

Reply:
sure can clean them up then, will do